### ISSUE TITLE
Création de procédures pour les EPCI sans fiscalité propre et sans compétence selon Banatic

### ISSUE THREAD

**Issue by HermanceGauth** [MTES-MCT/Docurba#935]
https://jam.dev/c/07f2136b-0d8e-4c50-9f4c-cb92b4ebbcf6

Sûrement car il n'est pas dans la liste des collectivités dans l'onglet Mes collectivités

Aujourd'hui, nous ajoutons les EPCI avec une procédure lorsque l'on génère les "référentiels".

Le travail ici consistera à :
- Ne plus ajouter les EPCI avec une procédure à la génération du référentiel
- Sur les pages où l'on ne souhaite lister que les EPCI "cherchables", on regardera : 
  - les EPCI avec compétence
  - les EPCI à fiscalité propre
  - les EPCI avec au moins une procédure

Endroit où l'on se contente de lister les EPCI "cherchables" :
- Page "Mes Collectivités"
- Page d'accueil
- Certains exports ?

**Comment by HermanceGauth** [comment]
J'ai bien le même soucis pour tous les SM qui ne sont pas dans la liste des collectivités

**Comment by HermanceGauth** [comment]
Attention, on reste sur le fait que dans l'onglet Mes collectivités on ne veut pas afficher tous les groupements "hors sujet" (n'ayant pas la compétence). Par contre il se peut que le PETR du Grand Verdun ne soit pas hors sujet mais que ce soit un pb similaire au SMF du Grand Nevers.

**Comment by UngererFabien** [comment]
Yes j'ai ce bug en tête, c'est les groupements de groupements qui bug parce qu'ils n'arrivent pas a trouver les communes.

**Comment by HermanceGauth** [comment]
Impossible à tester car je ne retrouve plus ce PETR dans la liste des collectivités cf mon message laissé ici : https://github.com/MTES-MCT/Docurba/issues/797

**Comment by UngererFabien** [comment]
@HermanceGauth je viens de re test en prod il y a bien le PETR et tout s'affiche. Probablement un décalage entre les pull request. Je me souviens qu'il y avait une PR pour l'affichage dans la liste qui utilisait l'ancienne API et une PR pour la création de SCOT pour les groupement de groupement.

Je te laisse re test ;)

**Comment by HermanceGauth** [comment]
@UngererFabien peux-tu juste me réexpliquer pourquoi ce PETR s'affiche dans la liste lorsque l'on fait le parcours "Mes Procédures" > "Ajouter une procédure" et pas dans le tableau de bord de l'onglet "Mes collectivités"?

**Comment by HermanceGauth** [comment]
J'ai pu effectivement ouvrir le formulaire pour le PETR du Grand Verdun, je ne suis pas allée jusqu'à créé la procédure pour ne pas perturber l'enquête de la DDT, je pourrai retester plus tard mais c'était un cas théorique, il n'a pas de SCoT à l'échelle de ce PETR aujourd'hui _a priori_.

**Comment by HermanceGauth** [comment]
@UngererFabien pour ne pas perturber l'enquête dans le 55, j'ai choisi un département qui n'utilise pas Docurba (36) pour tester l'ajout de procédure sur un groupement non identifié comme ayant la compétence Banatic.
Mon parcours connecté DDT :
- Onglet "Mes procédures"
- "Nouvelle procédure"
- Je choisis un syndicat qui n'est pas visible dans l'onglet "Mes collectivités", ici : Syndicat d'aménagement du bassin de l'Indre 
- J'accède bien au formulaire de création de procédure
- Quand je clique sur "Créer la procédure" la page ne charge pas comme si la création avait échouée
- Si je retourne sur l'onglet "Mes procédures" la procédure que je viens de créé apparaît bien
- Si je clique dessus depuis cette liste, elle ne charge pas comme lors de la création
- La procédure apparaît bien dans l'onglet "Mes collectivités" colonne SCoT des communes concernées
- L'EPCI n'est pas présent dans la liste "Mes collectivités"

Attendu : permettre la création de procédure sur ces EPCI sans fiscalité propre jusqu'à l'affichage de la FRP et la possibilité de la compléter (a minima) et que cet EPCI apparaisse dans l'onglet "Mes collectivités" (dans l'idéal)

**Comment by HermanceGauth** [comment]
Re testé ce jour en prod. La procédure s'ouvre bien et apparaît sur la page "Mes collectivités".

**Comment by HermanceGauth** [comment]
En revanche après avoir supprimé mon test, le SM qui ne devrait plus apparaître dans l'onglet "Mes collectivités" reste présent, j'ouvre une issue à part, je pense que ce ne sera pas un problème à l'avenir car très peu d'utilisateurs devraient faire ce parcours d'ajout et de suppression d'EPCI sans fiscalité propre et sans compétence banatic.

![Image](https://github.com/user-attachments/assets/667e7fef-5155-47bb-a8f1-7cce97724c25)

**Comment by rik** [comment]
J'ai listé le travail dans le commentaire initial.
